.rubocop.yml

```diff
@@ -139,11 +136,75 @@ Style/DateTime:
     - "rakelib/evss.rake"
     - "lib/evss/auth_headers.rb" #DateTime is needed for date formatting here
 
-
 # Skipping for now, should revisit:
 Rails/HasManyOrHasOneDependent:
   Exclude:
     - "app/models/education_benefits_claim.rb"
     - "app/models/saved_claim/education_benefits.rb"
     - "app/models/saved_claim.rb"
     - "app/models/terms_and_conditions.rb"
+
+# Using .empty? with the safe navigation operator seems to be the correct call in this situation
+Lint/SafeNavigationWithEmpty:
+  Exclude:
+    - 'lib/hca/configuration.rb'
+
+# The suggestions here don't seem to make sense in most cases. Skip for now.
+Naming/MemoizedInstanceVariableName:
+  Enabled: false
+
+# The suggestions here appear that they would break intended behavior. Skip for now.
+Naming/RescuedExceptionsVariableName:
```

Comment:
How can this break intended behavior?
Don't we just need to rename the rescued exception variable to `e` in each case to satisfy this cop?

Reply:
This was a total mistake on my part.